presqt/api_v1/views/resource.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from presqt.api_v1.helpers.function_router import FunctionRouter
+from presqt.api_v1.serializers.resource import ResourcesSerializer
+
+
+class ResourcesList(APIView):
+    """
+    **Supported HTTP Methods**
+
+    * Get: Retrieve a summary of all resources for the given Target.
```

Comment:
Perhaps: 
    * Get: Retrieve a summary of all resources for the given Target that a user has access to.